sklearn/cluster/k_means_.py

```diff
@@ -792,6 +792,22 @@ class KMeans(BaseEstimator, ClusterMixin, TransformerMixin):
     inertia_ : float
         Sum of distances of samples to their closest cluster center.
 
+    Examples
+    --------
+
+    >>> from sklearn.cluster import KMeans
+    >>> import numpy as np
+    >>> X = np.array([[1, 2], [1, 3], [1, 0],
+    ...               [4, 3], [4, 4], [5, 1]])
+    >>> kmeans = KMeans(n_clusters=2, random_state=0).fit(X)
+    >>> kmeans.labels_
+    array([1, 1, 1, 0, 0, 0], dtype=int32)
+    >>> kmeans.predict([[0, 0], [4, 4]])
+    array([1, 0], dtype=int32)
+    >>> kmeans.cluster_centers_
+    array([[ 4.33333333,  2.66666667],
+           [ 1.        ,  1.66666667]])
```

Comment:
Maybe you can change the values of the data points to get fractional number with finite number of digits such as `4.25` instead of `4.3333333`. This should make the doctest slightly more readable and less sensitive to a change of the precision of the display of the numpy arrays.